SECURITY.md

```diff
@@ -0,0 +1,23 @@
+## Security
+
+We do our best to make sure our products are free of security vulnerabilities. To reduce the risk of introducing a vulnerability,
+you can follow these best practices:
+
+* Always use the latest release. For security purposes, we sign our releases published on [Maven Central](https://central.sonatype.com/search?q=g:org.jetbrains.skiko)
```

Comment:
```suggestion
* Always use the latest release. For security purposes, we sign our releases published on Maven Central
```

- it points to skiko
- Compose has multiple groups and it seems we can't provide one link